src/Core/src/Fonts/FontManager.iOS.cs

```diff
@@ -41,7 +41,7 @@ public FontManager(IFontRegistrar fontRegistrar, ILogger<FontManager>? logger =
 		public UIFont GetFont(Font font, double defaultFontSize = 0) =>
 			GetFont(font, defaultFontSize, CreateFont);
 
-		public double GetFontSize(Font font, double defaultFontSize = 0) =>
+		double GetFontSize(Font font, double defaultFontSize = 0) =>
```

Comment:
The only thing that was calling this currently was inside FontManager so it didn't seem worthwhile to expose this